sinatra/.rubocop.yml

```
@@ -0,0 +1,26 @@
+
+# Ignore warnings about methods being too long
+Metrics/MethodLength:
+  Enabled: false
+
+# Ignore warnings about assignment, branch, and condition size
+Metrics/AbcSize:
+  Enabled: false
+
+# Ignore warnings about classes being too long
+Metrics/ClassLength:
+  Enabled: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)